test/testUtils/CurrencyFormatterSpec.scala

```diff
@@ -36,6 +36,23 @@ class CurrencyFormatterSpec extends TestSupport with ImplicitCurrencyFormatter {
       }
     }
 
+    "CurrencyFormatterInt.toNegativeCurrency" should {
+      "return a value as negative" when {
+        "the number is more than 0" in {
+          val amount: Int = 1
```

Comment:
* not sure why we use Int to model amount. maybe BigDecimal?, minor thing;
* to be 100% sure I would use property base testing to check if this conversion work for all variation of inputs(this is optional);

Reply:
We had two implicits for bigDecimal and Int, we've added tests for the Int and now have just pushed for the BigDecimal as well. You're right, property based tests are the best but for this case its should be sufficient (Andrii) Thank you!

Reply:
Pushed big decimal tests, is it ok if Andrew merges?

Reply:
yeh, sure